modules/mobile/app/helpers/mobile/facilities_helper.rb

```diff
@@ -18,19 +18,18 @@ def facilities_service
     end
 
     def address_from_facility(facility)
-      address = facility.address['physical']
-      return nil unless address
```

Comment:
This nil return actually seems problematic because we only use this function for filling out locations, which require an address object.

Reply:
yes, I think this needed to be changed outside this issue. I bet the address has just never been nil before now.

Reply:
So I tested this in the command line and indeed you can't create a location without an address. So this should be safe to remove.